database/dummyData/seedData.js

```diff
@@ -0,0 +1,57 @@
+const ratioDB = require('../ratios/index.js');
+
+let totalEvents;
+
+let createdAt = new Date('2017-07-14 15:37:42.997-07');
+const incrementTime = date => (new Date(date.getTime() + 8000));
+
+const generateUser = () => {
+  const userId = Math.floor(Math.random() * (totalEvents / 10));
+  let ratio = Math.random();
+  while (ratio < 0.2) {
+    ratio = Math.random();
+  }
+  return {
+    groupId: userId % 2,
+    userId,
+    ratio,
+    createdAt,
+  };
+};
+
+
+const generateUsers = () => {
+  const users = [];
+  for (let i = 1; i <= totalEvents; i += 1) {
+    createdAt = incrementTime(createdAt);
+    users.push(generateUser());
+  }
+  return users;
+};
+
+const addRows = (tableName, array) => (
+  ratioDB[tableName].bulkCreate(array)
+);
+
+const genDailyTotals = () => {
```

Comment:
Good to know you can use SQL functions to calculate average ratios by group per day. :)